src/functions/sendCandidateResults/framework/handler.ts

```diff
@@ -1,92 +1,14 @@
-import bottleneck from 'bottleneck';
-import { sendEmail } from '../application/service/send-email';
-import { DocumentsServiceError } from '../domain/errors/documents-service-error';
-import { EmailPersonalisation, LetterPersonalisation } from '../domain/personalisation.model';
-import { sendLetter } from '../application/service/send-letter';
-import { INotifyClient } from '../domain/notify-client.interface';
 import { container } from './di/inversify.config';
 import { TYPES } from './di/types';
 import { getUploadBatch } from './__mocks__/get-upload-batch.mock';
 import { StandardCarTestCATBSchema } from '@dvsa/mes-test-schema/categories/B';
-import { getEmailTemplateId, getLetterTemplateId } from '../application/service/get-template-id';
-import { IConfigAdapter } from './adapter/config/config-adapter.interface';
+import { IRequestScheduler } from './request-scheduler';
 
 export async function handler() {
   // TODO -  Use Real Service + get batch size from config
-  const testResults: StandardCarTestCATBSchema []  = getUploadBatch(250);
+  const testResults: StandardCarTestCATBSchema []  = getUploadBatch(1);
```

Comment:
Is `1` intended here? Feels like a debugging change.

Reply:
Good spot, it was a debugging change.